qiita_pet/handlers/analysis_handlers.py

```diff
@@ -363,3 +363,20 @@ def validate_absolute_path(self, root, absolute_path):
                 root, absolute_path)
         else:
             raise QiitaPetAuthorizationError(user_id, absolute_path)
+
+
+class SelectedSamplesHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        # Format seldata to get study IDs for the processed data
+        seldata = defaultdict(dict)
```

Comment:
please, use `_` for the variable names (e.g. `sel_data`) this is PEP8 so I think this should be blocking.